src/components/Home.js

```diff
@@ -1,11 +1,68 @@
 import React from 'react'
-import Graphs from '../images/Graphs.png'
 import { Button } from 'react-bootstrap'
 import Avatar from 'react-avatar'
 import Carousel from 'react-multi-carousel'
 
+import {
+  Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend,  ComposedChart, ResponsiveContainer, Area
+} from 'recharts'
+
 const Home = () => {
 
+  const [category, setCategory] = React.useState(null)
+
+  const CustomTooltip = ({ active, payload, label }) => {
+    if (active) {
+      return (
+        <div className="custom-tooltip">
+          <h2 className='label-heading'>{label}</h2>
+          <p className="label-spent">{`${payload[0].name} : ${payload[0].value}`}</p>
```

Comment:
@tams2429 What happens if there are more than 2 items in the `payload` object?

Reply:
So from what I interpreted from the recharts library (https://recharts.org/en-US/api/Tooltip), the payload object always contains the items defined in our input data after the first item which gets taken into the label object. I.e. if input data = {name: Jan, Spent: 1700, Spend: 1600}, then payload object = {spent: 1700, Spend: 1600} and label = Jan? So in our case the payload object does have 2 items and can have more? (i think line 20 I am referring to the second item in payload?)